### ISSUE TITLE
flask_patch : RuntimeError: dictionary changed size during iteration

### ISSUE THREAD

**Issue by myaspm** [pgjones/quart#25]
Hi,

I'm using quart in our application and it works successfully. But if i run the quart application on docker , it returns the following error. I could not find anything about the error. How can i fix this error ?

```
seffaflik_1  | Traceback (most recent call last):
seffaflik_1  |   File "run.py", line 2, in <module>
seffaflik_1  |     from app import create_app
seffaflik_1  |   File "/code/app/__init__.py", line 2, in <module>
seffaflik_1  |     from .app import create_app
seffaflik_1  |   File "/code/app/app.py", line 7, in <module>
seffaflik_1  |     from quart import Quart,flask_patch
seffaflik_1  |   File "/usr/local/lib/python3.6/site-packages/quart/flask_patch/__init__.py", line 5, in <module>
seffaflik_1  |     patch_all()
seffaflik_1  |   File "/usr/local/lib/python3.6/site-packages/quart/flask_patch/_patch.py", line 83, in patch_all
seffaflik_1  |     _patch_modules()
seffaflik_1  |   File "/usr/local/lib/python3.6/site-packages/quart/flask_patch/_patch.py", line 64, in _patch_modules
seffaflik_1  |     for name, module in sys.modules.items():
seffaflik_1  | RuntimeError: dictionary changed size during iteration
seffaflik_seffaflik_1 exited with code 1
```

and code blocks : 

```
from quart import Quart,flask_patch
from quart import (
    Blueprint,
    session,
    request,
    render_template,
    jsonify
)

def create_app(config=None, app_name=None, blueprints=None):

    if app_name is None:
        app_name = Config.Config.PROJECT

    if blueprints is None:
        blueprints = DEFAULT_BLUEPRINTS

    app = Quart(app_name)
    configure_app(app, config)
    configure_hook(app)
    configure_logging(app)
    configure_blueprints(app, blueprints)
    configure_extensions(app)
    configure_error_handlers(app)
    configure_filters(app)
    return app

```

**Comment by pgjones** [comment]
Hello, I think the issue is that you have combined imports in `from quart import Quart,flask_patch`, rather I think you should have `import quart.flask_patch` first before any other imports. I've hopefully made this clear in these [docs](https://pgjones.gitlab.io/quart/flask_extensions.html).

**Comment by myaspm** [comment]
Hello Again,

I'm following your process steps but It's not change anything. I moved `app=Quart("test")` to first row processed in run.py. 
It works succesfully to default but it's return again the same error. 

New code block :

```
# coding:utf-8


from quart import flask_patch
from quart import (
    Quart,
    Blueprint,
    session,
    request,
    render_template,
    jsonify
)
app = Quart("test")

from app import create_app
app = create_app(app=app)

app.run(host='0.0.0.0', port=5001, threaded=True, debug=True)
```
and  same error
```
Recreating seffaflik_seffaflik_1 ... done
Attaching to seffaflik_seffaflik_1
seffaflik_1  | Traceback (most recent call last):
seffaflik_1  |   File "run.py", line 4, in <module>
seffaflik_1  |     from quart import flask_patch
seffaflik_1  |   File "/usr/local/lib/python3.6/site-packages/quart/flask_patch/__init__.py", line 5, in <module>
seffaflik_1  |     patch_all()
seffaflik_1  |   File "/usr/local/lib/python3.6/site-packages/quart/flask_patch/_patch.py", line 83, in patch_all
seffaflik_1  |     _patch_modules()
seffaflik_1  |   File "/usr/local/lib/python3.6/site-packages/quart/flask_patch/_patch.py", line 64, in _patch_modules
seffaflik_1  |     for name, module in sys.modules.items():
seffaflik_1  | RuntimeError: dictionary changed size during iteration
seffaflik_seffaflik_1 exited with code 1
```

**Comment by pgjones** [comment]
Hmm, I can't seem to reproduce this. Is this the main file or is it imported from another? In addition is `import quart.flask_patch` the very first line (bar comments) in the file? Also can you try with `import quart.flask_path` rather than the from version?

**Comment by myaspm** [comment]
Yes this code is main file. I'm running this code with python run.py.  And I changed from quart import flask_patch the import quart.flask_patch but I get the same error. When I look at the flask extensions you support ,  I use flask-mongoengine library you do not support. Actually I work in normal mode without any trouble, but I get an error on the docker.And I import this mongoengine after import quart.flask_patch.
And there is one more thing. If I delete import quart.flask_patch on default mode then I get this following error

```
Traceback (most recent call last):
  File "run.py", line 6, in <module>
    app = create_app()
  File "/home/hanbey/workspaces/python/seffaflik/app/app.py", line 51, in create_app
    configure_extensions(app)
  File "/home/hanbey/workspaces/python/seffaflik/app/app.py", line 74, in configure_extensions
    db.init_app(app)
  File "/home/hanbey/.local/share/virtualenvs/seffaflik-2oDz6vDS/lib/python3.6/site-packages/flask_mongoengine/__init__.py", line 111, in init_app
    raise Exception('Invalid Flask application instance')
Exception: Invalid Flask application instance
```
So I can not delete import quart.flask_patch.

**Comment by pgjones** [comment]
I think this should be fixed with 0c64f5028c1e579d1f82e0d9ea5c2cf4ca3aa827, could you try it?

**Comment by pgjones** [comment]
I'm going to assume this is fixed in the latest release, 0.6.5, please reopen if it isn't.
